### PR TITLE
[WEBSITE] DataFusion 26.0.0 Blog Post

### PR DESCRIPTION
**Pull request by alamb**
Closes https://github.com/apache/arrow-datafusion/issues/5812

This PR tries to summarize the last 6 months of DataFusion and tell people what has been going on in the community.

### CODE REVIEW

File: _posts/2023-06-01-datafusion-25.0.0.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+[Various] benchmarks show DataFusion to be quite close or [even
+faster] to the state of the art in analytic performance (at the moment
+this seems to be DuckDB). We continually work on improving performance
+(see [#5546] for a list) and would love additional help in this area.
+
+[various]: https://voltrondata.com/resources/speeds-and-feeds-hardware-and-software-matter
+[even faster]: https://github.com/tustvold/access-log-bench
+[#5546]: https://github.com/apache/arrow-datafusion/issues/5546
+
+DataFusion now read single large Parquet files significantly faster by
```
Comment (paddyhoran) [inline _posts/2023-06-01-datafusion-25.0.0.md:103]:
```suggestion
DataFusion now reads single large Parquet files significantly faster by
```

File: _posts/2023-06-01-datafusion-25.0.0.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+DataFusion can run many different subqueries by rewriting them to
+joins. It has been able to run the full suite of TPC-H queries for at
+least the last year, but recently we have implemented significant
+improvements to this logic, sufficient to run almost all queries in
+the TPC-DS benchmark as well.
+
+## Community and Project Growth
+
+The six months since [our last update] saw significant growth in
+the DataFusion community .Between versions `17.0.0` and `26.0.0`,
```
Comment (paddyhoran) [inline _posts/2023-06-01-datafusion-25.0.0.md:258]:
```suggestion
the DataFusion community. Between versions `17.0.0` and `26.0.0`,
```

File: _posts/2023-06-01-datafusion-25.0.0.md [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+
+$ git shortlog -sn 17.0.0..26.0.0 . | wc -l
+      107
+-->
+
+# How to Get Involved
+
+Kudos to everyone in the community who has contributed ideas,
+discussions, bug reports, documentation and code. It is exciting to be
+innvating on the next generation of database architectures together!
```
Comment (paddyhoran) [inline _posts/2023-06-01-datafusion-25.0.0.md:291]:
```suggestion
innovating on the next generation of database architectures together!
```

**Review by paddyhoran — APPROVED** [review]

**Comment by ozankabak** [comment]
It'd be nice to re-run the Clickbench after all the perf work and put a link inside the article.

File: _posts/2023-06-01-datafusion-25.0.0.md [hunk 4]
```diff
@@ -0,0 +1,303 @@
+---
+layout: post
+title: "Apache Arrow DataFusion 25.0.0"
+date: "2023-06-02 00:00:00"
```
Comment (Folyd) [inline _posts/2023-06-01-datafusion-25.0.0.md:4]:
```suggestion
title: "Apache Arrow DataFusion 26.0.0"
date: "2023-06-20 00:00:00"
```

**Comment by alamb** [comment]
https://arrow.apache.org/blog/2023/06/24/datafusion-25.0.0/

**Comment by Folyd** [comment]
The url should be  [arrow.apache.org/blog/2023/06/24/datafusion-26.0.0](https://arrow.apache.org/blog/2023/06/24/datafusion-26.0.0/)?

**Comment by alamb** [comment]
> The url should be [arrow.apache.org/blog/2023/06/24/datafusion-26.0.0](https://arrow.apache.org/blog/2023/06/24/datafusion-26.0.0/)?

🤦  -- that is a good point. The URL probably should be different, but now that this blog has been published I would prefer not to fix it to avoid breaking the links

**Pull request merged**